src/app/launcher/create-app/targetenvironment-createapp-step/target-environment-createapp-step.component.html

```diff
@@ -53,7 +54,7 @@ <h3 class="card-pf-body--body_header">Benefits</h3>
         <button class="btn btn-primary"
                 [disabled]="stepCompleted !== true"
                 (click)="navToNextStep()">Continue</button>
-        <button class="btn btn-link" (click)="launcherComponent.cancel()">Cancel</button>
+        <button class="btn btn-default" (click)="launcherComponent.cancel()">Cancel</button>
```

Comment:
Please change back to `.btn-link`, per design

Reply:
@mindreeper2420 Hi Adam. We are following this UX link for design reference - https://redhat.invisionapp.com/share/6BEZL73M7#/screens/275860134

According to this design, 'Cancel' appears in the button format.

If you have any other design reference for this, please share it so that we can follow them as well.